lib/internal/test_runner/reporter/dot.js

```diff
@@ -3,19 +3,19 @@ const {
   ArrayPrototypePush,
   MathMax,
 } = primordials;
-const colors = require('internal/util/colors');
 const { formatTestReport } = require('internal/test_runner/reporter/utils');
+const { red, white, green, clear } = require('internal/util/colors');
```

Comment:
please avoid destructuring here so `colors.refresh` will still work

Reply:
@MoLow thanks for the super prompt response :)

I applied the code change, but I don't see any reference to `colors.refresh`, can you pinpoint me to the right place?